src/targets/gpu/compile_hip_code_object.cpp

```diff
@@ -153,29 +153,33 @@ std::size_t compute_block_size(std::size_t n, std::size_t max_block_size)
     return std::min(std::max(min_block_size, block_size), max_block_size);
 }
 
+std::vector<src_file> migraphx_kernels()
```

Comment:
Dont overload the function name.

Reply:
But name it something else to avoid ambiguity. Why is this even needed for windows?